.github/workflows/release.yml

```diff
@@ -84,18 +84,14 @@ jobs:
           external_repository: backpack/android
           publish_branch: main
 
-      - name: Publish Release APK to AppCenter
-        uses: wzieba/AppCenter-Github-Action@v1.3.4
-        with:
-          appName: Backpack/Backpack-Android
-          token: ${{secrets.APP_CENTER_TOKEN}}
-          group: Everyone
-          file: app/build/outputs/apk/oss/release/app-${{ env.flavour }}-release.apk
-          releaseNotes: Version ${{github.event.release.tag_name}}
-
-  SupernovaPublish:
-    runs-on: ubuntu-22.04
-    environment: Publishing
-    steps:
       - name: Update Supernova docs
```

Comment:
removed the separate pipeline for the supernova docs. this means we'll only get prompted once about approving the release